dist/js/game.js

```diff
@@ -36,24 +57,58 @@ function gameLoop(delta){
 }
 
 function add_characther(x, y, img_filepath){
```

Comment:
```suggestion
function add_character(x, y, img_filepath){
```
Men även byta stavning överallt annars